### ISSUE TITLE
problem: adding a new blockchain project not as simple as possible

### ISSUE THREAD

**Issue by gazhayes** [Blockrazor/blockrazor#615]
Problem: the 'add currency' route could probably be simpler. I've opened this issue to start a discussion and see if it's worth doing anything about it. Please go and try to add a new currency so you get a feel for how it works and then come back here and share your criticisms and ideas :)

ping @samuelralak @AshHimself @sjmackenzie @janat08 @MadanBhandari @anbud @kevgathuku

**Comment by janat08** [comment]
the tooltip text is blurry on uhd screen.

required fields unmarked.

some field should be focused to demonstrate the tooltips.

unresponsive.

select fields don't expand on focus with tab (don't know if they actually do).

features section goes empty if everything is unchecked, but title remains

if you select all options it looks bad.

**Comment by janat08** [comment]
My suggestion going way back to use aside instead of tooltips not that it's any more responsive.

**Comment by gazhayes** [comment]
> features section goes empty if everything is unchecked, but title remains
> required fields unmarked.
> if you select all options it looks bad.

Yep, good catch.

> the tooltip text is blurry on uhd screen.
> unresponsive.

This page will basically only be used by people on an actual computer. 

> some field should be focused to demonstrate the tooltips.

Can you explain more?

**Comment by AshHimself** [comment]
@gazhayes - I seriously think we should investigate an another more UX friendly datepicker. I did find another that wasn't purely for bootstrap and it does look easy to use. Maintained repo too https://github.com/xdan/datetimepicker

**Comment by gazhayes** [comment]
> we should investigate an another more UX friendly datepicker.

Just log it as a low priority issue for now to track it. I'm actually not sure what's more convenient anymore because datepickers usually (looks like this is no different) require a shitload of clicks when you want to go back a few years (and many of these projects go back almost 10 years).

**Comment by AshHimself** [comment]
@gazhayes  NLP date picker then :P https://sugarjs.com/

**Comment by janat08** [comment]
some field should be focused to demonstrate the tooltips.:
the fields are technical, and it's not evident that there're tooltips for them before user decides to not add currency.

**Comment by anbud** [comment]
I'm not sure about changing the date picker, this one works perfectly fine, and it's usually quite hard to navigate through years on a regular date picker.

If you select all three checkboxes on the first part of the form, the UI becomes a little bit weird:
![img](http://i.imgur.com/sMJZYVB.png)

The error handling part needs a little bit of work, but there's already an issue for that I believe.

Basically, other than that, it looks perfectly fine, at least for me. Maybe the tooltips could be a little bit bigger.
